source/retryable-writes/retryable-writes.rst

```diff
@@ -216,72 +216,84 @@ This error label can be found in the top-level "errorLabels" field of the error.
 In a server error response with a writeConcernError field the top level document
 or the writeConcernError document may contain the RetryableWriteError error label.
 
+RetryableWriteError Labels
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
 The RetryableWriteError label might be added to an error in a variety of ways:
 
-When the driver encounters a network error communicating with any server version
```

Comment:
This section was a bit hard to scan, so I broke it down to a bulleted list, which I found easier to read, most of the text is unchanged. Let me know if this new organization seems okay.